client/src/Movies/SavedList.js

```diff
@@ -7,13 +9,15 @@ export default class SavedList extends Component {
 
   render() {
     return (
+      <Link to="/">
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
```

Comment:
Do you feel like your starting to get a good handle for escaping the JSX context back to javascript?